provisioning/common/etcd/values.yaml

```diff
@@ -62,7 +62,7 @@ auth:
     allowNoneAuthentication: false
   token:
     type: jwt
-    ttl: 10m
+    ttl: 10080m # temporary: https://github.com/etcd-io/etcd/pull/14995
```

Comment:
V `etcd-client` - `3.5.6` sa pridavalo nejake watch retry, ked token expiroval, ... v `3.5.7` bol revert.
https://github.com/etcd-io/etcd/pull/14995 

My sme mali doteraz nastavene rotacie tokenu kazdych `10min`.
Docasne to nastavujem na `7dni`, kym sa mi to podari nastudovat, nasimulovat a vyriesit.

Reply:
Vyzera to tak, ze worker sa po rotacii JWT tokenu nevie z toho dostat.

Reply:
Vysledkom je zmena ENV:
![image](https://user-images.githubusercontent.com/19371734/214550421-efd449fe-a914-4f4b-9e80-32654b160847.png)